.github/workflows/linux-workflow.yml

```
@@ -45,22 +45,22 @@ jobs:
             # Need to disable PCH until cmake 3.17
             # (PCH conflicts with ccache, fixed by https://gitlab.kitware.com/cmake/cmake/-/merge_requests/4400)
             cmakeflags: -DCMAKE_DISABLE_PRECOMPILE_HEADERS=ON
-            detail: -nopch
+            detail: " nopch"
             experimental: false
           - os: ubuntu-20.04
             platform: x86
             compiler: gcc
             cmakeflags: -DCMAKE_DISABLE_PRECOMPILE_HEADERS=ON
-            detail: -nopch
+            detail: " nopch"
             experimental: false
           - os: ubuntu-20.04
             platform: x86
             compiler: gcc
             cmakeflags: -DCMAKE_DISABLE_PRECOMPILE_HEADERS=ON -DARCH_FLAG=-march=haswell
-            detail: -avx2-nopch
+            detail: " avx2 nopch"
             experimental: false
 
-    name: "${{ matrix.platform }} | ${{ matrix.compiler }} ${{ matrix.detail }}"
+    name: "${{ matrix.platform }} | ${{ matrix.compiler }}${{ matrix.detail }}"
```

This doesn't need quotation marks.
```suggestion
    name: ${{ matrix.platform }} | ${{ matrix.compiler }}${{ matrix.detail }}
```